pkg/permissions/actions.go

```diff
@@ -14,31 +14,31 @@ var (
 )
 
 const (
-	ReadRepositoryAction   = "fs:ReadRepository"
-	CreateRepositoryAction = "fs:CreateRepository"
-	AttachStorageNamespace = "fs:AttachStorageNamespace"
-	ImportFromStorage      = "fs:ImportFromStorage"
-	CancelImport           = "fs:ImportCancel"
-	DeleteRepositoryAction = "fs:DeleteRepository"
-	ListRepositoriesAction = "fs:ListRepositories"
-	ReadObjectAction       = "fs:ReadObject"
-	WriteObjectAction      = "fs:WriteObject"
-	DeleteObjectAction     = "fs:DeleteObject"
-	ListObjectsAction      = "fs:ListObjects"
-	CreateCommitAction     = "fs:CreateCommit"
-	CreateMetaRangeAction  = "fs:CreateMetaRange"
-	ReadCommitAction       = "fs:ReadCommit"
-	ListCommitsAction      = "fs:ListCommits"
-	CreateBranchAction     = "fs:CreateBranch"
-	DeleteBranchAction     = "fs:DeleteBranch"
-	ReadBranchAction       = "fs:ReadBranch"
-	RevertBranchAction     = "fs:RevertBranch"
-	ListBranchesAction     = "fs:ListBranches"
-	CreateTagAction        = "fs:CreateTag"
-	DeleteTagAction        = "fs:DeleteTag"
-	ReadTagAction          = "fs:ReadTag"
-	ListTagsAction         = "fs:ListTags"
-	ReadConfig             = "fs:ReadConfig"
+	ReadRepositoryAction    = "fs:ReadRepository"
+	CreateRepositoryAction  = "fs:CreateRepository"
+	AttachStorageNamespace  = "fs:AttachStorageNamespace"
+	ImportFromStorageAction = "fs:ImportFromStorage"
+	CancelImportAction      = "fs:ImportCancel"
```

Comment:
- CancelImportAction -> ImportCancelAction
- can you also align ReadConfig and AttachStorageNamespace